extensions/amp-ad-network-adsense-impl/0.1/responsive-state.js

```diff
@@ -149,6 +186,27 @@ export class ResponsiveState {
     return /** @type {!ResponsiveState} */ (state);
   }
 
+  /**
+   * Convert the element to container width responsive.
+   *
+   * @param {!Element} element
+   * @return {!ResponsiveState} responsive state
+   * @private
+   */
+  static fallBackToContainerWidth_(element) {
```

Comment:
I find the usage of fallback here confusing, really we are changing the element to be container width right? Maybe something like `upgradeToContainerWidth`?

Reply:
I think a lot of this would be less confusing with this change, something like `isContainerWidth()` instead of `isFallback` etc.

Reply:
FYI, we decided to make the container-width the "The fall back state" for the Full-With Responsive Tags. I just rename them to make it more straightforward.

Reply:
Can check out go/amp-responsive-summary for more details.